packages/webpack-cli/__tests__/ConfigGroup/webpack.promise.config.cjs

```diff
@@ -0,0 +1,13 @@
+module.exports = () => {
```

Comment:
cjs uses export default

Reply:
That's for mjs, cjs uses module.exports AFAIU

Reply:
Yes, `export default` is `mjs`

Reply:
ah right, could we add one for mjs too?

Reply:
mjs support is WIP here - https://github.com/webpack/webpack-cli/pull/1728

Will add for mjs after I finish that work 😄